### PR TITLE
[skip ci] update to latest release of harbor

### PR DESCRIPTION
**Pull request by mhagen-vmware**

### CODE REVIEW

**Comment by rajanashok** [comment]
lgtm

File: tests/resources/Util.robot [hunk 1]
```diff
@@ -445,9 +445,9 @@ Run Docker Info
 
 Install Harbor To Test Server
     [Arguments]  ${user}=%{TEST_USERNAME}  ${password}=%{TEST_PASSWORD}  ${host}=%{TEST_URL}  ${datastore}=${TEST_DATASTORE}  ${network}=%{BRIDGE_NETWORK}  ${name}=harbor
-    ${out}=  Run  wget https://github.com/vmware/harbor/releases/download/0.4.1/harbor_0.4.1_beta.ova
-    ${out}=  Run  ovftool harbor_0.4.1_beta.ova harbor_0.4.1_beta.ovf
-    ${out}=  Run  ovftool --datastore=${datastore} --name=${name} --net:"Network 1"="${network}" --diskMode=thin --powerOn --X:waitForIp --X:injectOvfEnv --X:enableHiddenProperties --prop:vami.domain.Harbor=mgmt.local --prop:vami.searchpath.Harbor=mgmt.local --prop:vami.DNS.Harbor=8.8.8.8 --prop:vm.vmname=Harbor harbor_0.4.1_beta.ovf 'vi://${user}:${password}@${host}'
+    ${out}=  Run  wget https://github.com/vmware/harbor/releases/download/0.4.5/harbor_0.4.5_beta.ova
```
Comment (mlh78750) [inline tests/resources/Util.robot]: They had to update something for the TLS cert install and I think one other thing and issued a respin. Find it at: https://github.com/vmware/harbor/releases/download/0.4.5/harbor_0.4.5_beta_respin2.ova

**Comment by mlh78750** [comment]
I added an update for the "latest" .4.5 harbor link. After that good to go..

LGTM

**Comment by mlh78750** [comment]
LGTM

**Pull request merged**